molli/chem/bond.py

```diff
@@ -755,6 +760,11 @@ def append_bonds(self, *bonds: Bond) -> None:
         for b in bonds:
             b.parent = self
 
+            if b.a1 not in self.atoms:
+                self.append_atom(b.a1)
+            if b.a2 not in self.atoms:
+                self.append_atom(b.a2)
+
```

Comment:
I think that we could use a simpler syntax that would also avoid some repetitions:
```python
def append_bonds(self, *bonds):
    map(self.append_bond, bonds)
```